src/test/k6/tests/find-tests.js

```diff
@@ -0,0 +1,38 @@
+import http from 'k6/http';
+import { browser } from 'k6/browser';
+import { sleep } from 'k6';
+
+export async function searchResultsTest() {
+    const page = await browser.newPage();
+
+    try {
+        await page.goto('https://test.find-support-for-your-family.education.gov.uk/');
+
+        const startButton = await page.locator('input[class="govuk-button govuk-button--start"]').click();
+
+        await Promise.all([page.waitForNavigation(), startButton.click()]);
+        
+        await page.locator('input[name="postcode"]').type('E1 2EN');
+
+        const submitButton = await page.locator('input[type="button"]');
+
+        await Promise.all([page.waitForNavigation(), submitButton.click()]);
+
+        const headerText = await page.locator('h1').textContent();
+        check(headerText, {
+            header: headerText === 'Your local family hubs, services and activities',
+        }); //title attribute on the head rather than content body.
+        
+    } finally {
+        await page.close();
+    }
+    sleep(1);
+}
+
+export function verifyStatusCodeTest() {
+    const res = http.get('https://test.find-support-for-your-family.education.gov.uk/ServiceFilter?postcode=E1%202EN&adminarea=E09000030&latitude=51.517612&longitude=-0.056838&frompostcodesearch=True');
```

Comment:
As above, this should probably be a config (passed in as a command line argument?), that way we can specify an environment to run against such as `local`, `test` or `test2`, and then in the code the URL can be calculated from the input.